ddl/backfilling.go

```diff
@@ -769,6 +767,15 @@ func (b *backfillScheduler) adjustWorkerSize() error {
 		case typeCleanUpIndexWorker:
 			idxWorker := newCleanUpIndexWorker(sessCtx, i, b.tbl, b.decodeColMap, reorgInfo, jc)
 			worker, runner = idxWorker, idxWorker.backfillWorker
+		case typeReorgPartitionWorker:
+			partWorker, err := newReorgPartitionWorker(sessCtx, i, b.tbl, b.decodeColMap, reorgInfo, jc)
+			if err != nil {
+				if b.canSkipError(err) {
+					continue
+				}
+				return err
+			}
```

Comment:
The errors returned from `newReorgPartitionWorker` are not recoverable... We had better not to skip them :)

Reply:
OK, I agree and have removed it. But I don't think it would be too bad, since it would just limit the number of workers if by somehow the `newReorgPartitionWorker` would have succeeded.

Reply:
When `newReorgPartitionWorker` is possible to succeed for part of workers, we can add it back and update the log message inside `canSkipError`.